core/math/expression.cpp

```diff
@@ -69,6 +69,7 @@ const char *Expression::func_name[Expression::FUNC_MAX] = {
 	"inverse_lerp",
 	"range_lerp",
 	"smoothstep",
+	"move_to",
```

Comment:
why move_to instead of move_towards here?

Reply:
I did try to use move_towards but I had an issue with the name having conflicts in the code editor with the Vector version. The editor would confuse Vector.move_towards with the core move_towards, using the wrong syntax highlighting and providing the wrong documentation for it.

Reply:
We should not design the API to workaround script editor bugs IMO, so it should be `move_towards`. Other functions in this class also have the same name as their scripting API counterparts.

Reply:
I'll change it back to move_towards. I'll try to see if I can fix the bug but I'm guessing it won't be trivial.